src/Basket.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 
-const Basket = () => {
-  return (
-    <section>
-      <p>Shopping Cart</p>
-      <ul>
-        <li>Phone 1 <button>x</button></li>
-        <li>Phone 2 <button>x</button></li>
-        <li>Phone 3 <button>x</button></li>
-      </ul>
-    </section>
-  );
-};
+class Basket extends React.Component {
+    render() {
+       return (
+        <section>
+          <p>Shopping Cart</p>
+          <ul>
+              {this.props.basketItems.map((item,remId) => {
+                const itemCount = (item.count === 1) ? '' : (' (' + item.count + ')' );
```

Comment:
```suggestion
                const itemCount = (item.count === 1) ? '' : `(${item.count})`;
```
Template string is powerful